.changes/2176-show-suggested.md

```diff
@@ -0,0 +1 @@
+- Suggested spaces and chats are preferably now shown on the overview of a space, if configured
```

Comment:
I had a impression that suggestions can automatically show up based on the relevant interactions of rooms and don't need to populate manually. Is this how the sdk supports it OR we intend to choose this way?

Reply:
Not entirely sure I get your question. The way that "suggested" work is that it is a flag in the `child`-room event, marking specific `children` spaces as `suggested`. this is explicitly done by the parent room admin and allows them to curate the list of spaces/chats to be shown and suggest to the user...